helm-chart/amalthea/values.yaml

```diff
@@ -127,13 +128,47 @@ crdNames:
   shortNames:
     - js
 
-
 # Determine how often amalthea checks each server to determine whether it is idle.
 # The criteria for deeming that a server is idle are based on the cpu usage
-# and the stats provided by the jupyter server /api/status endpoint. The culling 
+# and the stats provided by the jupyter server /api/status endpoint. The culling
 # section of the custom resource definition of JupyterServer can be used to control
 # on a per-server basis whether a server is ever culled and also if it is culled then
 # how long each server is idle for before it is culled.
 culling:
   jupyterServerIdleCheckIntervalSeconds: 300
   cpuUsageMillicoresIdleThreshold: 500
+
+# Deploy an optional additional scheduler in the same namespace where the operator is deployed.
+# The additional scheduler allows for fine grained scheduling policies for the pods managed by this operator.
+# As an example the pods can be scheduled to prioritize the most requested nodes, thus "packing" the workload in
+# as few nodes as possible, instead of spreading them over all the available nodes.
+# NOTE: ensure that the tag of the scheduler image matches the version of the Kubernetes cluster where it is deployed.
+scheduler:
+  enable: false
+  image:
+    repository: k8s.gcr.io/kube-scheduler
+    pullPolicy: IfNotPresent
+    tag: v1.22.1
+  priorities:
+    - name: MostRequestedPriority
+      weight: 100
+    - name: ServiceSpreadingPriority
+      weight: 1
+    - name: EqualPriority
+      weight: 1
+    - name: ImageLocalityPriority
+      weight: 1
+    - name: SelectorSpreadPriority
+      weight: 1
+    - name: InterPodAffinityPriority
+      weight: 1
+    - name: LeastRequestedPriority
+      weight: 1
+    - name: BalancedResourceAllocation
+      weight: 1
+    - name: NodePreferAvoidPodsPriority
+      weight: 1
+    - name: NodeAffinityPriority
+      weight: 1
+    - name: TaintTolerationPriority
```

Comment:
Does that mean that our taints/tolerations with which we currently force Jupyter servers onto certain nodes would be overruled by the dense packing policy?

Reply:
No: they coexists, thus the scheduling policy only applies to the nodes that still satisfy the taints/tolerations/affinity declared by the deployments.

Reply:
👍 perfect!